wouso/resources/templates/cpanel/races/all.html

```diff
@@ -22,45 +22,160 @@
 {% endblock %}
 
 {% block sectioncontent %}
-<ul>
-    {% for r in races %}
-    <li>
-        {{ r }} {% if r.can_play %}[can play]{% endif %}
-        <a href="{% url admin:user_race_change r.id %}"
-            style="color: green" title="Edit">
-            <span class="glyphicon glyphicon-edit"></span>
-        </a>
-        <ul>
-            {% for g in r.children %}
-            <li>
-	        {{ g.name}} {{ g }}
-                <a href="{% url admin:user_playergroup_change g.id %}"
-                   style="color: green" title="Edit">
-                    <span class="glyphicon glyphicon-edit"></span>
+<div style="text-align: right;">
+    <button class="btn btn-sm btn-default" style="margin-bottom: 10px;" id="toggle_groups" data-show="1">
+        Show all groups
+    </button>
+</div>
+
+<table class="table table-hover table-bordered table-condensed">
+    <thead>
+        <th>#</th>
+        <th>Name</th>
+        <th>Title</th>
+        <th>Can play</th>
+        <th style="width: 250px;">Actions</th>
+    </thead>
+    <tbody>
+        {% for r in races %}
+        <tr>
+            <td style="font-weight: bold;">{{ forloop.counter0 }}</td>
+            <td>{{ r.name }}</td>
+            <td>{{ r.title }}</td>
+            <td>
+                {% if r.can_play %}
+                    <h3 style="margin: 0px;"><span class="label label-success label-sm">can play</span></h3>
```

Comment:
Replace "can play" to "Yes". That's what we're using in similar tables across CPanel (e.g. Spells)